### PR TITLE
[CF-475] Add handling oh HTTP 504 to glance.

### PR DESCRIPTION
**Pull request by antonf**
Add handling of HTTP 504 to glance migration code.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by antonf** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry/lib/os/image/glance_image.py [hunk 1]
```diff
@@ -293,12 +293,14 @@ def get_ref_image(self, image_id):
                 glance_exceptions.NotFound,
                 glance_exceptions.HTTPInternalServerError,
                 ssl.ZeroReturnError,
+                glance_exceptions.HTTPException,
```
Comment (SVilgelm) [inline cloudferry/lib/os/image/glance_image.py]: Remove `glance_exceptions.NotFound` and `glance_exceptions.HTTPInternalServerError`, `HTTPException` is a base class to them.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1

**Comment by SVilgelm** [comment]
👍 can be merged

**Comment by archyufa** [comment]
+1 tested in staging

**Pull request merged**